kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java

```diff
@@ -317,6 +281,23 @@ public RootPaths rootPaths() {
   }
 
   @Override
+  public String getPodLog(String namespace, String podName, String containerName, boolean pretty) {
```

Comment:
tried adding it there first but got loads of compile errors as pods().withNamespace(x).withName(y) doesn't return PodOperations so there's no way to add extra pod specific operations?